docs/FAQ.md

```diff
@@ -37,6 +37,23 @@ See [revsets] and [templates] for further guidance.
 
 Use `jj log -r ..`. The `..` [operator] lists all visible commits in the repo, excluding the root (which is never interesting and is shared by all repos).
 
+### Can I monitor how `jj log` evolves?
+
+The simplest way to monitor how the history as shown by `jj log` evolves is by using the `watch(1)` command (or [hwatch](https://github.com/blacknon/hwatch) or [viddy](https://github.com/sachaos/viddy)).
+For example:
+
+```sh
+watch jj --ignore-working-copy log --color=always
+```
+
+This will continuously update the (colored) log output in the terminal.
+The `--ignore-working-copy` option avoids conflicts with manual operations during the creation of snapshots.
+Martin used this in a [tmux](https://github.com/tmux/tmux/wiki) pane during his presentation [Jujutsu - A Git-compatible VCS](https://www.youtube.com/watch?v=LV0JzI8IcCY).
```

Comment:
Minor: I'd put a newline before this sentence and say `watch` instead of "this".

Reply:
Ok fixed, I have not added the newline though, I think it makes sense to keep the watch related paragraph in once piece.